NuGet.config

```diff
@@ -4,6 +4,7 @@
   <packageSources>
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
+    <add key="darc-pub-dotnet-core-setup-9a599ff" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-core-setup-9a599ffe/nuget/v3/index.json" />
```

Comment:
Don't need the changes in this file

Reply:
I have updated according to your request, please review. thanks